docs/src/modules/java-protobuf/pages/views.adoc

```diff
@@ -125,7 +125,7 @@ When no result is found, the request fails with gRPC status code `NOT_FOUND`. A
 
 The View state corresponding to an Entity is not automatically deleted when the Entity is deleted.
 
-When the source of the `eventing.in` is a Value Entity the `handle_deletes: true` annotation of `(kalix.method).eventing.in` defines that this method is used for handling Value Entity deletes. The method must have `google.protobuf.Empty` as a parameter. Kalix will automatically delete the View state, but you can also write your own delete handler when `transform_updates` flag is `true`. If you don't provide such method then, by default, your View state will not be deleted.
```

Comment:
The last sentence was removed, because is a duplication of the previous paragraph.